src/solver/variable/economy/bindingConstraintsMarginalCost.h

```diff
@@ -44,7 +44,7 @@ struct VCardBindingConstMarginCost
     //! Caption
     static const char* Caption()
     {
-        return "MARG. COST by BC";
+        return "BC. MARG. COST";
```

Comment:
Since it's obvious from the filename that it's a binding constraint, I suggest "MARG. COST"

Reply:
As far as I know, the function **Caption()** (in the case of our new output variable) is not used when writing reports.
It could be called (to be checked) form GUI to make the list of variables to be printed in the **Thematic trimming**.
In this case, we need to keep it that way to distinguish it from any other marginal price output variable.